assets/sass/widgets/_googlesitekit-widget-analyticsAllTrafficV2.scss

```diff
@@ -88,8 +88,9 @@
 	.googlesitekit-widget--analyticsAllTrafficV2__totalcount {
```

Comment:
Now that this issue has clarified the requirement of matching the style for these numbers and labels to be similar to our `DataBlock` styles, I think it's rather counter-productive to redeclare all of those styles here, since this results in duplicate code, and furthermore the styles used here don't _exactly_ match because e.g. responsive behavior is slightly different.

I think it would be better to reuse `DataBlock` CSS classes in the `TotalUserCount` component, since pretty much all pieces in there are similar. Only a single design tweak here requires extra CSS, which is the percentage value should also be in red/green according to the change (contrary to using the regular gray).

Reusing the classes clarifies the relation here, and it ensures even in the future these numbers and labels will be formatted consistently.